README.md

```diff
@@ -83,7 +83,7 @@ Where:
 ### Get the current user
 
 ```kotlin
-omgAPIClient.getCurrentUser().enqueue(object: Callback<OMGResponse<User>>{
+omgAPIClient.getCurrentUser().enqueue(object: Callback<User>{
```

Comment:
You forgot to rename `Callback` to `OMGCallback` in the readme :)

Reply:
Fixed it :)